cloudferrylib/os/storage/cinder_database.py

```diff
@@ -0,0 +1,188 @@
+from cloudferrylib.base import storage
+from cloudferrylib.utils import utils as utl
+from cloudferrylib.utils import mysql_connector
+from cinderclient.v1 import client as cinder_client
+import jsondate
```

Comment:
1. Please make sure we really need this extra dependency, this library seem to be not supported anymore (it was not updated since Nov 2012), maybe we can utilize standard python `json` library with some enhancements.
2. In case we really need this library, please add it to `requirements.txt`